test/node-and-browser.js

```diff
@@ -176,7 +176,6 @@ describe('node and browser', function () {
       return dataShouldEql();
     });
   };
-
```

Comment:
There is whitespace modified in this file, please undo this.

Reply:
Also reverted.

Reply:
It looks like you removed a newline here, please restore this.